Source/driver/meth_params.F90

```diff
@@ -801,8 +886,6 @@ subroutine ca_set_castro_method_params() bind(C, name="ca_set_castro_method_para
        gravity_type_int = 1
     else if (gravity_type == "PoissonGrav") then
        gravity_type_int = 2
-    else if (gravity_type == "PrescribedGrav") then
-       gravity_type_int = 3
```

Comment:
why is this removed?

Reply:
I'm not sure---I had made a change to `parse_castro_params.py`, and re-ran it with the change. Perhaps I didn't provide correct arguments to the script?

Reply:
Re-run the mk_params.sh script which invokves parse_castro_params.py for you.

Reply:
okay... maybe someone made the change directly to that file and not the template in `development`.  Let me take a look.

Reply:
> Re-run the mk_params.sh script which invokves parse_castro_params.py for you.

That's what I did initially.

Reply:
I updated the meth_params.template very recently, so perhaps you just need to sync again with development.